content/tracing/faq/how-long-is-tracing-data-stored.md

```diff
@@ -3,4 +3,37 @@ title: How long is tracing data stored?
 kind: faq
 ---
 
-Traces are stored for up to 6 months. The storage time for each trace is determined probabilistically. If you view a trace, it is guaranteed to be stored for 6 months.
\ No newline at end of file
+Individual traces are stored for up to 6 months.  
+
+To determine how long a particular trace will be stored, the Agent makes a sampling decision early in the trace's lifetime. In Datadog backend, sampled traces are retained according to time buckets:
+
+
+|Retention bucket|% of stream kept|
```

Comment:
I think we can remove the table; the example below feels clearer to me.

Reply:
I would let it like this, even if the text below gives more info the table acts as a nice summary.